packages/editor/src/components/post-author/index.js

```diff
@@ -95,6 +115,29 @@ function PostAuthor() {
 		return null;
 	}
 
+	if ( ! isCombobox ) {
+		const selectId = 'post-author-selector-' + instanceId;
+
+		return (
+			<PostAuthorCheck>
+				<label htmlFor={ selectId }>{ __( 'Author' ) }</label>
+				<select
+					id={ selectId }
+					defaultValue={ postAuthor.id }
+					onBlur={ handleSelect }
```

Comment:
Previously this was `onChange`, linter complained about that not being accessible so switched to `onBlur` and using `defaultValue` vs `value`.